Created Login and Signup Forms

emiliearenkid opened the pull request:
They are literally the most basic things and they took me too long to make for how basic they are. Also added bootstrap for formatting stuff in the App.js page.

emiliearenkid committed:
Created Login and Signup Forms

They are literally the most basic things and I am embarrassed by how long they took me to make. Also added bootstrap for formatting stuff in the App.js page.

emiliearenkid committed:
Merge branch 'main' into auth-interface

KristoferKwan commented on frontend/src/components/signupForm.js:
```diff
@@ -0,0 +1,144 @@
+                               onChange={handleChange}
+                        />
+                    </div>
+                    <div className="form-group text-left">
```
take out company from user registration -- taken out of model for simplicity

emiliearenkid committed:
Fixed!

Merge conflicts should be resolved and there shouldn't be any errors. Deleted the double import statement in App.js and added style-components to the frontend folder

emiliearenkid committed:
Removed Fields

Changed forms to fit backend schema so they only input their name, email, and password for registering.

KristoferKwan commented on .idea/MITR-Project.iml:
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
could you remove the .idea folder from the branch? not sure how to do it exactly with desktop, but will require u to add .idea into the gitignore and remove the .idea folder from the repository. (run the following command in the root folder of MITR-Project to remove .idea from the branch) `git rm --cached  -r .idea`.

KristoferKwan commented on frontend/src/App.js:
```diff
@@ -15,10 +17,12 @@ function App() {
           </Switch>
         </div>
+        <link rel="stylesheet"
+                href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
```
can you place this into index.html inside of the public folder?

emiliearenkid committed:
Moved stylesheet from App.js to index.js

emiliearenkid committed:
Got rid of .idea

Added the file to the .gitignore and removed the file.

KristoferKwan reviewed (approved):
looks good. My only comment is that the stylesheet link should be placed in the header of the react pages (which can be accessed through the index.html file in the public folder), but as we're eventually moving towards using material ui, this is fine for now^
